orcl-db-cicd-for-devs/workshops/freetier/manifest.json

```diff
@@ -1,31 +1,46 @@
 {
-  "workshoptitle": "Oracle Database CI/CD for Developers",
+  "workshoptitle": "Take control of your database with automated schema changes",
     "help": "livelabs-help-db_us@oracle.com",
 	"tutorials": [
         {
             "title": "Introduction",
-            "description": "Oracle Database CI/CD for Developers",
+            "description": "Take control of your database with automated schema changes",
             "filename": "../../intro/intro.md"
         },
         {
-            "title": "Getting Started",
-            "description": "Get a Free Trial",
-            "filename": "https://oracle-livelabs.github.io/common/labs/cloud-login/pre-register-free-tier-account.md"
+            "title": "Get started",
+            "description": "Get started - Cloud login",
+            "filename": "https://oracle-livelabs.github.io/common/labs/cloud-login/cloud-login.md"
         },
         {
-            "title": "Setups",
-            "description": "Setup your environment for this lab",
-            "filename": "../../setups/setups.md"
+            "title": "Create your autonomous database",
+            "description": "Create an autonomous database for use in the workshop",
+            "filename": "../../adb-provision-conditional/adb-provision-conditional.md"
         },
         {
-            "title": "Lab 1: Use SQLcl for Database Change Management/Tracking",
-            "description": "Use SQLcl for Database Change Management/Tracking",
-            "filename": "../../sqlcl/sqlcl.md"
+            "title": "Setup",
+            "description": "Set up your environment for this workshop",
+            "filename": "../../setup/setup.md"
         },
         {
-            "title": "Lab 2: Create and Apply Database Changes with SQLcl",
-            "description": "Create and Apply Database Changes with SQLcl",
-            "filename": "../../change/change.md"
+            "title": "Lab 1: Liquibase basics - help, generate, and update commands",
+            "description": "Become familiar with the core Liquibase commands",
+            "filename": "../../lab1-help-generate-update/lab1-help-generate-update.md"
+        },
+        {
+            "title": "(Optional) Lab 2: Get under the hood - understanding changelog tables & changelogs",
```

Comment:
(Optional) Lab 2 title: Get under the hood - understanding changelog tables & changelogs, uses a gerund. Consider using an imperative verb lab title instead of a gerund, for example, 'Start' instead of 'Starting'.